bundles/admin/admin-layereditor/view/LayerWizard.jsx

```diff
@@ -44,9 +53,10 @@ function getStep (layer) {
 const LayerTypeTitle = ({ layer, LabelComponent }) => (
     <React.Fragment>
         <Message messageKey='wizard.type' LabelComponent={LabelComponent} />
-        { layer.type && `: ${layer.type}` }
+        { layer.type && <React.Fragment><div style={{ display: 'inline' }}>:</div><div>{layer.type}</div></React.Fragment>}
```

Comment:
`<span>` instead of `<div style={{ display: 'inline' }}>`?

Reply:
Fixed to use span as suggested